Scripts/Swift/SetUpDeliveryConstants.swift

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/swift
+
+import Foundation
+
+let fileManager = FileManager.default
+
+print("Do you want to set up Constants values? (Can be edited later) [Y/n]: ")
+
+var arg = "y"
+
+arg = readLine() ?? arg
```

Comment:
Could this be `let arg = readLine() ?? "y"`?

Reply:
Updated https://github.com/nimblehq/ios-templates/pull/496/commits/d5bf8197b64aabd8fe16808ca480c48a6dd4ba01